cypress/test-apps/js/package.json

```diff
@@ -22,7 +22,7 @@
     "search-insights": "1.7.1"
   },
   "devDependencies": {
-    "parcel": "2.0.1"
+    "parcel": "2.0.0-beta.2"
```

Comment:
Is it possible to bump all of the packages out of beta instead of reverting?

Reply:
Ironically the stable versions don't work with M1.